README.md

```diff
@@ -12,86 +12,160 @@ Read more in: [What is this repo?](#what-is-this-repo)
+  - [Sample App CR and ConfigMap for the Control Plane](#sample-app-cr-and-configmap-for-the-control-plane)
+  - [Important note about required cluster level config](#important-note-about-required-cluster-level-config)
+- [Configuration Options](#configuration-options)
 - [Limitations](#limitations)
 - [For developers](#for-developers)
   - [Installing the Chart locally](#installing-the-chart-locally)
   - [Release Process](#release-process)
 - [What is this repo?](#what-is-this-repo)
 
 
-# Usage
+# Installing
 
+There are 3 ways to install this app onto a tenant cluster.
 
+1. [Using our web interface](https://docs.giantswarm.io/reference/web-interface/app-catalog/)
+2. [Using our API](https://docs.giantswarm.io/api/#operation/createClusterAppV5)
+3. Directly creating the App custom resource on the Control Plane.
 
-## Sample values files
+## Sample values files for the web interface and API
 
 This is an example of the values file you could upload using our web interface.
 
 ```
 # values.yaml
 
-app: hello
-yolo: bolo
+configmap:
+  error-log-level: info
 ```
 
-If you are not using the web interface, our (deprecated) API takes the same structure but formatted as JSON:
+If you are not using the web interface, our (deprecated) API takes the same
+structure but formatted as JSON:
 
 ```
 # values.json
 
 {
-  "app": "hello",
-  "yolo": "bolo",
+  "configmap": {
+    "error-log-level": "info"
+  }
 }
 ```
 
-## Sample App CR and ConfigMap
+## Sample App CR and ConfigMap for the Control Plane
 
-If you have access to the Kubernetes API on the Control Plane, you could create the App CR and ConfigMap directly. Here is an example:
+If you have access to the Kubernetes API on the Control Plane, you could create
+the App CR and ConfigMap directly. Here is an example:
 
 ```
 # appCR.yaml
-
-WIP
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 1.0.0
+  name: nginx-ingress-controller-app
+  namespace: abc12
+
+spec:
+  name: nginx-ingress-controller-app
+  namespace: kube-system
+  catalog: giantswarm
+  version: 1.6.8
+
+  userConfig:
+    configMap:
+      name: nginx-ingress-controller-user-values
+      namespace: abc12
+    secret:
+      name: ""
+      namespace: ""
+
+  config:
+    configMap:
+      name: ingress-controller-values
+      namespace: abc12
+    secret:
+      name: ""
+      namespace: ""
+
+  kubeConfig:
+    context:
+      name: abc12-kubeconfig
+    inCluster: false
+    secret:
+      name: abc12-kubeconfig
+      namespace: abc12
 ```
 
+## Important note about required cluster level config
+
+The `ingress-controller-values` ConfigMap referenced in the `spec.config` field of the App CR
+is required for the ingress controller to work properly.
+
+`ingress-controller-values` is created by our operators and it helps set values unique to your tenant cluster. When creating this App using our web interface or our API,
```

Comment:
```suggestion
The `ingress-controller-values` ConfigMap referenced in the `spec.config` field of the App CR
is required for the ingress controller to work properly. `ingress-controller-values` is created by our operators and it helps set values unique to your tenant cluster. When creating this App using our web interface or our API,
```

Dont see why to have a paragraph break

Reply:
I find the sentence really important, so I want it to have whitespace around it so that eye's can catch it at a glance.

> The `ingress-controller-values` ConfigMap referenced in the `spec.config` field of the App CR is required for the ingress controller to work properly.